index.html

```diff
@@ -101,7 +96,252 @@ <h1>I'm Adeyemi Glad to see you!</h1>
+                      <div class="Css_2">
+                        <p>css</p>
+                      </div>
+                    </li>
+                  </div>
+                  <div class="textTagjava_2">
+                    <li>
+                      <div class="Java_2">
+                        <p>javascript</p>
+                      </div>
+                    </li>
+                  </div>
+                </ul>
+            </div>
+          </div>
+          <section class="cardWork_card_3">
+            <img id="snap_portfolio_3">
+            <div class="leftBlock_3">
+              <div class="pryTxt_head_3">
+                <div class="text_head_3">
+                  Multi-Post
+                  Stories
+                </div>
+                <div class="frame_card_3">
+                  <div class="Client_card_3">
+                    <div class="Client_txt_3">
+                      CANOPY
+                    </div>
+                  </div>
+                  <span class="Counter_3"></span>
+                  <div class="Role_card_3">
+                    <div class="Role_txt_3">
+                      Back End Dev
+                    </div>
+                  </div>
+                  <span class="Counter2_3"></span>
+                  <div class="Year_card_3">
+                    <p class="Year_txt_3">
+                      2015
+                    </p>
+                  </div>
+                </div>
+              </div>
+              <div class="pryText_body_3">
+                <p>A daily selection of privately personalized reads; no accounts or sign-ups required.
+                </p>
+              </div>
+              <ul class="Tags_3">
+                <div class="textTag_Hml_3">
+                  <li >
+                    <div class="Hml_3">
+                      <p>html</p>
+                      </div>
+                  </li>
+                </div>
+                <div class="textTagcss_3">
+                  <li>
+                    <div class="Css_3">
+                      <p>css</p>
+                    </div>
+                  </li>
+                </div>
+                <div class="textTagjava_3">
+                  <li>
+                    <div class="java_3">
+                      <p>javascript</p>
+                      </div>
+                  </li>
+                </div>
+              </ul>
+              <div id="Action_bttn_3">
+                <div class="projbttn_3">
+                  <div class="click_bttn_3">
+                  <button href="#SeeProject" >See Project</button>
+                </div>
+                </div>
+              </div>
+            </div>
+          </section>
+          <section class="cardWork_4">
+            <img id="snap_portfolio_4"/>
+            <div class="leftBlock_4">
+              <div class="pryTxt_head_4">
+                <div class="text_head_4">
+                  Multi-Post
+                  Stories
```

Comment:
Kindly put this in `h` tag. This is important in SEO search and also to draw in the reader and give a clear indication of the content on the page ☺